spp_farmer_registry_demo/__manifest__.py

```diff
@@ -20,6 +20,7 @@
         "spp_farmer_registry_default_ui",
         "spp_base_demo",
         "queue_job",
+        "spp_custom_filter_farmer_registry",
```

Comment:
Aside from `spp_custom_filter_farmer_registry`, you should also depend on `spp_custom_filter` since you are adding `allow_filter` in some of the fields in `spp_farmer_registry_demo`.